### DISCUSSION TITLE
Is it possible to restore about:preferences no search results illustration

### DISCUSSION THREAD

**Discussion by erwin4270** [black7375/Firefox-UI-Fix#654] [Q&A] [answered]
Before Firefox 92, if you search in the Settings (`about:preferences`) that contains no results, it will display an illustration.

I think I like it, but the illustration is removed in Firefox 92 and above.

Is it possible to restore this illustration?

**Screenshots**
Firefox 91
![91](https://user-images.githubusercontent.com/104198709/221219850-ba885a04-a96c-467c-952d-da2f308b51d6.png)
Firefox 92+
![92](https://user-images.githubusercontent.com/104198709/221219877-bd86d88f-a536-4fec-a590-a7bd471f1e7a.png)

**Comment by black7375** [comment, marked as answer]
That's a good idea.
I will add it to the next version. https://github.com/black7375/Firefox-UI-Fix/commit/60e60f98c0137bd4bc9a3126570400a9b58f8e3b
